config/environments/production.rb

```diff
@@ -94,5 +94,5 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_mailer.default_url_options = { :host => 'https://initiative-tracker-stagi-pr-22.herokuapp.com' }
+  config.action_mailer.default_url_options = { :host => 'https://initiative-tracker-stagi-pr-23.herokuapp.com' }
```

Comment:
I think we need to look for a different way of doing this so that we don't need a commit to change the value. Maybe there's an environment variable on Heroku that has the value?

Reply:
I have added .env file to get dynamic heroku staging app name.